src/styles/notifications.less

```diff
@@ -263,4 +263,112 @@ ux-alert {
         background-color: @alert-dark-bg;
         border: 1px solid @alert-dark-border;
     }
-}
\ No newline at end of file
+}
+
+/*
+ * Badge
+ */
+
+.ux-badge-container {
+    position: relative;
+    overflow: visible;
```

Comment:
There might be components which use overflow: hidden to e.g. truncate content text when a max-width is specified, in which case this would get overridden. I don't think there's any way of automatically resolving that conflict, so probably we would have to add a note to the documentation recommending to use a wrapper element in this situation.